buildah.go

```diff
@@ -117,6 +118,9 @@ type BuilderOptions struct {
 	// ReportWriter is an io.Writer which will be used to log the reading
 	// of the source image from a registry, if we end up pulling the image.
 	ReportWriter io.Writer
+	// github.com/containers/image/types SystemContext to hold credentials
+	// for use with private repositories
+	CredsContext *types.SystemContext
```

Comment:
I've changed this so SystemContext as suggested.   I was trying to not use that as we've a systemContext in play elsewhere.  However I was thinking about possibly merging that in to this at a later time, might try that now to avoid confusion.